demo/python/leopard_demo_file.py

```diff
@@ -17,16 +17,31 @@
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('--access_key', required=True)
-    parser.add_argument('--library_path', default=None)
     parser.add_argument('--model_path', default=None)
+    parser.add_argument('--library_path', default=None)
+    parser.add_argument('--enable_automatic_punctuation', action='store_true')
```

Comment:
I believe we decided that punctuation should enabled by default in the demos, meaning that the flag is to disable it